library/psa_crypto_driver_wrappers.c

```diff
@@ -77,21 +77,19 @@
 #define PSA_CRYPTO_MBED_TLS_DRIVER_ID (1)
 #define MBEDTLS_TEST_OPAQUE_DRIVER_ID (2)
 #define MBEDTLS_TEST_TRANSPARENT_DRIVER_ID (3)
-
-/* END-driver id */
```

Comment:
Update version in peaks/data/pack_definition_files/pack_arm_mbed_tls.xml also needs to be updated to 3.4.0+renesas.2